### DISCUSSION TITLE
APACHE-GUACAMOLE : RDP Connection

### DISCUSSION THREAD

**Discussion by sannier3** [community-scripts/ProxmoxVE#967] [Q&A]
Hello,

First of all, thank you for making the installation of Apache Guacamole much easier with your script!

I am encountering an issue with RDP connections. Whenever I try to initiate an RDP session, I get an error that seems to indicate an internal issue, but I am unsure how to resolve it.

Here are some details about the setup and the issue:

Environment: Proxmox : Debian 12 LXC (created by the community script for Apache Guacamole).
Error Message: An internal error has occurred within the Guacamole server, and the connection has been terminated. If the problem persists, please notify your system administrator, or check your system logs. Reconnecting in ...
Steps Taken: I reviewed multiple resources, and one link seemed particularly relevant: https://kifarunix.com/guacamole-how-to-fix-rdp-server-closed-refused-connection-security-negotiation-failed-wrong-security-type/
Could this issue be related to RDP configuration or security settings? For example, do I need to adjust the security mode or encryption settings on either the Guacamole server or the target VM?

Any guidance on how to debug or resolve this issue would be greatly appreciated. Let me know if I should provide additional logs or configuration details to help diagnose the problem.

Thank you for your support and for your work on this great project!

[ScriptInstallLogs.log](https://github.com/user-attachments/files/18220274/ScriptInstallLogs.log)
![image](https://github.com/user-attachments/assets/8ddb037e-eef7-46c0-b555-1ef391825f22)

**Comment by Logico68** [comment]
I had the same error and I solved it making this change in the guacd.conf file.

Edit the file: **nano /etc/guacamole/guacd.conf**
Modify bind_host value from 127.0.0.1 to 0.0.0.0: **bind_host = 0.0.0.0**
Save the file
Reboot

I hope it can be helpful.
